packages/panzer/disc-fe/src/Panzer_Workset.hpp

```diff
@@ -51,6 +51,7 @@
 
 #include "Panzer_Dimension.hpp"
 #include "Panzer_BasisValues2.hpp"
+#include "Panzer_PointValues2.hpp"
```

Comment:
This is a dumb question, but can we move these Values headers out of the workset header? It may not save us anything, but it would be nice to have a small, lightweight workset.